apply schema validation changes to all remaining fields, fixes #…

jacobthill opened the pull request:
## Why was this change made?
Need to convert all metadata fields with text to hashes in order to store the language of the metadata as a key.

## Was the documentation (README, API, wiki, ...) updated?
n/a

aaron-collier commented on traject_configs/bnf_common_config.rb:
```diff
@@ -61,4 +61,33 @@
+  'agg_provider',
+  'agg_provider_country',
+  'cho_alternative',
+  'cho_contributor',
```
I think this may be an issue with duplication, cho_contributor, cho_creator, etc, were not in bnf_common originally. Have you determined that the hash output is OK?

jacobthill commented:
@aaron-collier I'm not sure what changes you want? I fixed the duplication issues as far as I can tell. I fixed BnF and Stanford. Do you see other issues?
